joomla/joomla-cms
# Redo Isis flat repaint Installation (Closes #11917)

Pull Request for Issue #11917
### Summary of Changes

This is a rebased version of#11917 which looks broken against 3.7.x
### Testing Instructions

Same as #11917 -> Install joomla per: https://github.com/zero-24/joomla-cms/archive/11917.zip and confirm the new style
### Documentation Changes Required

None (screenshots maybe)
### Credits

The original creator is @ciar4n and many other see #11917
